### PR TITLE
Print title

### PR DESCRIPTION
**Pull request by johnnyvuanly**
@ClaraJames

### CODE REVIEW

File: guess_the_number.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import random
 
-print('Guess the Number')
+print('Guess the Number game')
```
Comment (claraj) [inline guess_the_number.py]: Can you make the G in game uppercase?

**Review by claraj — CHANGES_REQUESTED** [review]

File: guess_the_number.py (hunk 1 above)
Comment (johnnyvuanly) [reply guess_the_number.py]: Sure I can work on it

Comment (johnnyvuanly) [reply guess_the_number.py]: (Pretend we are tagging Clara) Completed the step for you!

Comment (claraj) [reply guess_the_number.py]: Looks great! thank you!

**Review by claraj — APPROVED** [review]

**Pull request merged**